.github/workflows/canary.yml

```diff
@@ -0,0 +1,49 @@
+name: canary
+on:
+  - pull_request
+  - push
+jobs:
+  canary:
+    name: testing ${{matrix.package}} with ${{matrix.node}} on ${{matrix.os}}
+    runs-on: ${{matrix.os}}
+    steps:
+      - name: checkout micromark
+        uses: actions/checkout@v2
+      - name: setup node
+        uses: dcodeIO/setup-node-nvm@master
+        with:
+          node-version: ${{matrix.node}}
+      - name: update npm
+        run: npm install -g npm
+      - name: install micromark
+        run: npm install
+      - name: build micromark
+        run: npm run build
+
+      - name: checkout ${{matrix.package}}
+        uses: actions/checkout@v2
+        with:
+          repository: ${{matrix.package}}
+          path: canary/${{matrix.package}}
+      - name: test ${{matrix.package}}
+        run: |
+          npm install
+          npx rimraf "**/micromark"
```

Comment:
Wouldn’t `npx` significantly slow down this process compared to having it (guessing we already have it?)

Reply:
We already have it, `npx` is allowing us to call it without needing to specify which local `node_modules` folder it is in.